src/main/java/com/github/webicitybrowser/spec/fetch/imp/BodyImp.java

```diff
@@ -0,0 +1,31 @@
+package com.github.webicitybrowser.spec.fetch.imp;
+
+import com.github.webicitybrowser.spec.fetch.Body;
+
+import java.io.InputStreamReader;
+
+public class BodyImp implements Body {
+
+	private InputStreamReader isr;
+	private byte[] source;
+	private int length;
+
+	public BodyImp(InputStreamReader isr, byte[] source, int length) {
+		this.isr = isr;
```

Comment:
`isr` is not a good field name. How about `bodyStream`?
And, to differentiate the byte[] from the reader, you can call it `bodyBytes`
The `length` field should not exist. InputStreamReaders communicate their length by returning a -1, and byte[] has its length as a field